src/sentry/static/sentry/app/styles/spacingScale.jsx

```diff
@@ -0,0 +1,28 @@
+export default size => {
+  if (size > 9) return (size - 9) * 100;
+
+  switch (size) {
```

Comment:
i'm using a case/switch here mostly because i started with an array and found myself writing out a long comment that looked exactly like a case switch...I'm fine with the array but we'll need something that makes it easy for people to figure out exactly what each number spits out